packages/desktop-client/src/components/sidebar/SidebarWithData.tsx

```diff
@@ -88,6 +88,7 @@ function EditableBudgetName({ prefs, savePrefs }: EditableBudgetNameProps) {
         type="bare"
         color={theme.buttonNormalBorder}
         style={{
+          color: theme.sidebarItemText,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why is this necessary?

Reply:
Removed